src/res_comments_table.ml

```diff
@@ -1682,6 +1682,14 @@ and walkExprArgument (_argLabel, expr) t comments =
         recordRows
         t
         comments
+    (* #462: Handle nested Ppat_or correctly *)
+    | Ppat_or ({ppat_desc = (Ppat_or(_, _))} as pattern1, pattern2) ->
```

Comment:
Since we're dealing with a list of things, it might be easier to take advantage of the helper function `walkList`.

Reply:
Done